cirq/ops/common_channels.py

```diff
@@ -227,33 +230,53 @@ def asymmetric_depolarize(p_x: Optional[float] = None,
 class DepolarizingChannel(gate_features.SingleQubitGate):
     """A channel that depolarizes a qubit."""
 
-    def __init__(self, p: float) -> None:
+    def __init__(self, p: float, n_qubits: int = 1) -> None:
         r"""The symmetric depolarizing channel.
 
-        This channel applies one of four disjoint possibilities: nothing (the
-        identity channel) or one of the three pauli gates. The disjoint
-        probabilities of the three gates are all the same, p / 3, and the
-        identity is done with probability 1 - p. The supplied probability
+        This channel applies one of 2**n disjoint possibilities: nothing (the
+        identity channel) or one of the 2**n - 1 pauli gates. The disjoint
+        probabilities of the Pauli gates are all the same, p / (2**n - 1), and
+        the identity is done with probability 1 - p. The supplied probability
         must be a valid probability or else this constructor will raise a
         ValueError.
 
         This channel evolves a density matrix via
 
             $$
             \rho \rightarrow (1 - p) \rho
-                    + (p / 3) X \rho X + (p / 3) Y \rho Y + (p / 3) Z \rho Z
+                + 1 / (2**n - 1) \sum _i P_i X P_i
             $$
 
+            where P_i are the 2**n - 1 Pauli gates (excluding the identity).
+
         Args:
             p: The probability that one of the Pauli gates is applied. Each of
-                the Pauli gates is applied independently with probability p / 3.
+                the Pauli gates is applied independently with probability
+                p / (2**n - 1).
+            n_qubits: the number of qubits.
 
         Raises:
             ValueError: if p is not a valid probability.
         """
 
+        error_probabilities = {}
+        all_identity = ''.join(['I'] * n_qubits)
```

Comment:
```python
all_identity = 'I' * n_qubits
```

Also, you don't need to set this in a variable - just inline in the conditional below.